### DISCUSSION TITLE
Novice questions :)

### DISCUSSION THREAD

**Discussion by stewie777** [Chia-Network/chia-blockchain#5275] [Support]
Hello everyone, I recently started farming chia and today, looking at the connections, I realized that I have connections to two wallets.
it's a normal thing?
from the screenshots it seems to you that everything is fine or something is not right?
thank you all :)
and good farming ... waiting for the pools :)
![Screenshot (4)](https://user-images.githubusercontent.com/1301893/118481561-e76eec00-b713-11eb-8de2-ed50c6527ebd.png)
![Screenshot (5)](https://user-images.githubusercontent.com/1301893/118481578-ec33a000-b713-11eb-81e8-402175a5e316.png)

**Comment by stewie777** [comment]
anyone can help me?

**Comment by lvcivs** [comment]
The first wallet is your own. 127.0.01 always points to localhost, that is your own computer. The second wallet is at a different ip, so it would appear to be someone else running only a wallet instead of a full node. This is normal and nothing to worry about.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
